lib/CapabilityInvocation.js

```diff
@@ -60,13 +66,24 @@ module.exports = class CapabilityInvocation extends ControllerProofPurpose {
       }
     }
     this.suite = suite;
+
+    if(currentDate && !(currentDate instanceof Date)) {
```

Comment:
Same issues as in CapabilityDelegation.

Reply:
Attempted to address in 32e102b746c3b6443e855917c5a9a51b34b83b9d